src/test/java/dev/kodice/hexcolorgenerator/models/dto/RandomColorsResponseTest.java

```diff
@@ -0,0 +1,30 @@
+package dev.kodice.hexcolorgenerator.models.dto;
+
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanConstructor;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanToString;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidGettersAndSetters;
+import static com.google.code.beanmatchers.BeanMatchers.registerValueGenerator;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+class RandomColorsResponseTest {
+
+	@Test
+	public void testRandomColorsResponse() {
+		registerValueGenerator(() -> {
+			double m = (Math.random() * (12)) + 1;
+			double d = (Math.random() * (25)) + 1;
+			return LocalDateTime.of(2019, (int) m, (int) d, (int) m, (int) d);
```

Comment:
Pudiéramos hacer variables separadas para la hora y los minutos?
```suggestion
			return LocalDateTime.of(2019, (int) month, (int) day, (int) hrs, (int) min);
```

Reply:
Listo ya acutalicé los nombres de las variables